lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php

```diff
@@ -115,6 +135,11 @@ class Config implements \Cm\RedisSession\Handler\ConfigInterface
      */
     const SESSION_MAX_LIFETIME = 31536000;
 
+    /**
+     * Try to break lock for at most this many seconds
+     */
+    const DEFAULT_FAIL_AFTER = 15;
```

Comment:
Why is this one not configurable? Comparing to `PARAM_BREAK_AFTER`?

Reply:
It's also necessary to update `setup:config:set` with new params (in `\Magento\Setup\Model\ConfigOptionsList\Session`).

Could you please also update http://devdocs.magento.com/guides/v2.2/config-guide/redis/redis-session.html ?

Reply:
I copied `DEFAULT_FAIL_AFTER` from 2.2-develop https://github.com/magento/magento2/blob/0839bd421bd5812e9aa7c943b89ecae9d5008832/lib/internal/Magento/Framework/Session/SaveHandler/Redis/Config.php#L121 Should I change it to be configurable here?

Thanks for pointing out the changes required to `setup:config:set`. I'll get that taken care of.

Reply:
Got it... ok, let's leave it as is.